src/link/Elf.zig

```diff
@@ -201,6 +202,21 @@ pub const TextBlock = struct {
     }
 };
 
+/// A list of text blocks in a specific section
+const TextBlockList = struct {
+    free_list: std.ArrayListUnmanaged(*TextBlock),
+    last_block: ?*TextBlock,
+    phdr_index: ?u16,
+    section_index: ?u16,
+
+    const empty = TextBlockList{
+        .free_list = .{},
+        .last_block = null,
+        .phdr_index = null,
+        .section_index = null,
+    };
```

Comment:
```suggestion
    free_list: std.ArrayListUnmanaged(*TextBlock) = .{},
    last_block: ?*TextBlock = null,
    phdr_index: ?u16 = null,
    section_index: ?u16 = null,
```
We have default values for this purpose, `TextBlockList.empty` can be replaced with `.{}`.

Reply:
Thank you for the review ! I think I did this because the `TextBlock` struct does the same, and just did not think it through.